generate_analysis_graphs.py

```diff
@@ -84,6 +87,53 @@
         individuals = TracedDataJsonIO.import_jsonl_to_traced_data_iterable(f)
     log.info(f"Loaded {len(individuals)} individuals")
 
+    # Compute the number of messages, activations and participants per episode and overall.
+    log.info("Computing the per-episode and per-season engagement counts...")
+    engagement_counts = OrderedDict()
+    for plan in PipelineConfiguration.RQA_CODING_PLANS:
+        engagement_counts[plan.dataset_name] = {
+            "Episode": plan.dataset_name,
+            "Total messages per show with Opt-ins": 0,
+            "Total activations per show with Opt-ins": 0,
+            "Total messages with Opt-ins": '-',
+            "Total participants with Opt-ins": '-'
+        }
+    engagement_counts["Total"] = {
+        "Episode": "Total",
+        "Total messages with Opt-ins": 0,
```

Comment:
Can you add '-' to the other two fields here, so we can signal that the cells intentionally don't contain numbers rather than be mysteriously left blank, just as you did for the per-week datasets

Reply:
updated thank you.